packages/icons/docs/stories/helpers/filterIconsBySearchTerm.js

```diff
@@ -0,0 +1,10 @@
+export default function filterCategoriesByName(data, query) {
```

Comment:
normally for consistency we name the function same as the file name, in this case the function name is `filterCategoriesByName` and the file name `filterIconsBySearchTerm`, I simply would name it filter